projects/storefrontlib/src/cms-components/product/product-list/container/product-list-component.service.spec.ts

```diff
@@ -132,7 +132,7 @@ describe('ProductListComponentService', () => {
         service.model$.subscribe();
 
         expect(productSearchService.search).toHaveBeenCalledWith(
-          ':relevance:category:testCategory',
+          ':relevance:allCategories:testCategory',
```

Comment:
we should drop the brand related test

Reply:
We still have the brand route until/unless other changes happen. This test is for the scenario where we have brandCode